RecoJets/JetPlusTracks/plugins/JetPlusTrackCorrector.cc

```diff
@@ -1129,12 +1364,42 @@ bool JetPlusTrackCorrector::matchElectrons(TrackRefs::const_iterator& itrk,
     }
   }
 
-  if (deltaR < 0.02)
+  if (deltaRMIN < 0.02)
     return true;
   else
     return false;
 }
 
+bool JetPlusTrackCorrector::matchElectrons(TrackRefs::const_iterator& itrk,
+                                           const edm::Handle<pat::ElectronCollection>& elecs) const {
+  if (elecs->empty()) {
+    return false;
+  }
+  // std::cout<<"JetPlusTrackCorrector::matchElectrons "<<std::endl;
+  double deltaR = 999.;
+  double deltaRMIN = 999.;
+  pat::ElectronCollection::const_iterator ielec = elecs->begin();
+  pat::ElectronCollection::const_iterator jelec = elecs->end();
+  for (; ielec != jelec; ++ielec) {
+    //  std::cout<<"Electron "<<ielec->eta()<<" "<<ielec->phi()<<" Track "<<(*itrk)->momentum().eta()<<" "<<
+    //     (*itrk)->momentum().phi()<<" PT "<<ielec->pt()<<" "<<(*itrk)->pt()<<std::endl;
+    double deltaphi = fabs(ielec->phi() - (*itrk)->momentum().phi());
+    if (deltaphi > 6.283185308)
```

Comment:
As above, I could suggest to use `M_PI`.

Reply:
change to deltaR.